src/dialogs/more-info/controls/more-info-fan.js

```diff
@@ -135,19 +156,36 @@ class MoreInfoFan extends LocalizeMixin(EventsMixin(PolymerElement)) {
   computeClassNames(stateObj) {
     return (
       "more-info-fan " +
-      attributeClassNames(stateObj, ["oscillating", "speed_list", "direction"])
+      (supportsFeature(stateObj, 1) ? "has-percentage " : "") +
```

Comment:
Please make this a const in `data/fan.ts`: `SUPPORT_PERCENTAGE = 1`

Reply:
There was no `data/fan.ts` so I created it and put the constants in it.